test/mocha/10-documents.js

```diff
@@ -30,240 +36,308 @@ const key2 = new X25519KeyAgreementKey2020({
+        error = e;
+      }
+      assertNoError(error);
 
-  it('should produce a 34 byte Buffer give a small value', async () => {
-    let result;
-    let error;
-    const value = '670dbcb1-164a-4d47-8d54-e3e89f5831f9';
-    try {
-      result = await documents._hmacString({hmac, value});
-    } catch(e) {
-      error = e;
-    }
-    assertNoError(error);
-    result.should.be.instanceOf(Buffer);
-    result.should.have.length(34);
-  });
+      result1.should.eql(result2);
+    });
 
-  it('should produce a 34 byte Buffer given a large value', async () => {
-    let result;
-    let error;
-    const value = crypto.randomBytes(4096).toString('hex');
-    try {
-      result = await documents._hmacString({hmac, value});
-    } catch(e) {
-      error = e;
-    }
-    assertNoError(error);
-    result.should.be.instanceOf(Buffer);
-    result.should.have.length(34);
-  });
+    it('should produce different output given different values', async () => {
+      let result1;
+      let error;
+      try {
+        result1 = await documents._hmacString({
+          hmac,
+          value: '294c9caa-707a-4758-ae5c-fe7306c25cc2'
+        });
+      } catch(e) {
+        error = e;
+      }
+      assertNoError(error);
 
-  it('should produce the same output given the same value', async () => {
-    let result1;
-    let error;
-    const value = '294c9caa-707a-4758-ae5c-fe7306c25cc2';
-    try {
-      result1 = await documents._hmacString({hmac, value});
-    } catch(e) {
-      error = e;
-    }
-    assertNoError(error);
-
-    let result2;
-    error = undefined;
-    try {
-      result2 = await documents._hmacString({hmac, value});
-    } catch(e) {
-      error = e;
-    }
-    assertNoError(error);
-
-    result1.should.eql(result2);
+      let result2;
+      error = undefined;
+      try {
+        result2 = await documents._hmacString({
+          hmac,
+          value: '0e26c923-84e6-4918-9337-f82c56951007'
+        });
+      } catch(e) {
+        error = e;
+      }
+      assertNoError(error);
+
+      result1.should.not.eql(result2);
+    });
   });
+});
 
-  it('should produce different output given different values', async () => {
-    let result1;
-    let error;
-    try {
-      result1 = await documents._hmacString({
-        hmac,
-        value: '294c9caa-707a-4758-ae5c-fe7306c25cc2'
+describe('Documents Database Tests', function() {
+  describe('Indexes', function() {
+    beforeEach(async () => {
+      const collectionName = 'tokenization-registration';
+      await cleanDB({collectionName});
+    });
+    it(`is NOT indexed for 'registration.internalId' in getRegistration()`,
+      async function() {
+      // Note: an index is not created for `registration.internalId` because
+      // queries based on that field alone are rare; they are not run in hot
+      // code paths nor do they need to run quickly -- so the index cost is not
+      // justified. Therefore, a collection scan should be done.
```

Comment:
```suggestion
        // Note: an index is not created for `registration.internalId` because
        // queries based on that field alone are rare; they are not run in hot
        // code paths nor do they need to run quickly -- so the index cost is not
        // justified. Therefore, a collection scan should be done.
```